### PR TITLE
create a central redirect for 'current Compliance Suite'

### PR DESCRIPTION
**Pull request by ge0rg**
It makes some sense to have a central link for the "current" Compliance Suite, that will get updated every year.

We have https://xmpp.org/about/compliance-suites.html as a description of the CS, so having https://xmpp.org/about/compliance-suites-current seemed logical to me.

### CODE REVIEW

**Comment by Echolon** [comment]
Please don't merge until #953 has been merged

**Comment by Echolon** [comment]
Hi @ge0rg ,

we have been upgrading from Pelican to the Hugo website generator. Could you review this in case it is still relevant?

That would be really great!

**Comment by ge0rg** [comment]
I've updated the commit for the new content directory structure.

**Review by mwild1 — APPROVED** [review]

**Pull request merged**
